CONTRIBUTING.md

```diff
@@ -0,0 +1,147 @@
+# Contributing
+
+This document explains how to build the Wasm module of the Internet Identity canister from scratch.
+
+> 💡 Binary releases are available on the [release page][releases].
+
+> 💡 The simplest way to build the code is to use the [Docker build][docker-build].
+
+> 💡 Make sure to read up on the [build features and flavors][features-and-flavors].
+
+The build requires the following dependencies:
+
+* `dfx` version 0.8.3
```

Comment:
0.9.2 works as well.

Reply:
ack! will leave 0.8.3 as long as it's what we use on CI.